docker/build/Dockerfile

```diff
@@ -42,8 +42,8 @@ ADD ./apache-dolphinscheduler-incubating-${VERSION}-dolphinscheduler-bin.tar.gz
 RUN mv /opt/apache-dolphinscheduler-incubating-${VERSION}-dolphinscheduler-bin/ /opt/dolphinscheduler/
 ENV DOLPHINSCHEDULER_HOME /opt/dolphinscheduler
 
-#4. install pg
-RUN apk add postgresql postgresql-contrib
+#4. install database
+RUN apk add postgresql postgresql-contrib mysql-client
```

Comment:
mysql's license can not meet the requirements of apache, pls remove it

Reply:
+1